website2/documentation/concepts.mdx

```diff
@@ -0,0 +1,153 @@
+export const info = {
+  name: "concepts",
+  url: "concepts",
```

Comment:
I like the fact that we use the file names for the url. I think because we're doing that we can remove these `url` properties, right?